packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx

```diff
@@ -230,6 +233,7 @@ export const ClayPaginationBarWithBasicItems = ({
 					<Picker
 						activeDelta={activeDelta}
 						aria-describedby={ariaDescribedby}
+						aria-label="Pagination"
```

Comment:
I think this should be `aria-label={labels.pagination}`

Reply:
Thanks @matuzalemsteles 
Fixed!